bin/crew

```diff
@@ -279,6 +279,10 @@ def search (pkgName, silent = false)
   rescue => e
     puts "Error with #{pkgName}.rb: #{e}".red unless e.to_s.include?('uninitialized constant')
```

Comment:
Can we use `.lightred` instead of `.red`?  It adds clarity to the text.

Reply:
changed